src/main/java/Duke.java

```diff
@@ -1,85 +1,20 @@
-import task.Task;
-import task.ToDo;
-import task.Event;
-import task.Deadline;
+import task.TaskList;
 
 import java.util.Scanner;
-import java.util.StringJoiner;
 
 public class Duke {
-    private static final Task[] taskList = new Task[100];
-    private static int taskCount = 0;
-
-    private static String getTaskCountString() {
-        return String.format("Now you have %d task%s in the list.", taskCount, taskCount == 1 ? "" : "s");
-    }
-
-    private static void print(String... strings) {
-        System.out.println(Messages.LINE.MESSAGE);
-        for (String s : strings) {
-            System.out.println(s);
-        }
-        System.out.println(Messages.LINE.MESSAGE);
-    }
-
-    private static void printTaskList() {
-        StringJoiner taskListString = new StringJoiner(System.lineSeparator());
-        for (int i = 0; i < taskCount; i++) {
-            taskListString.add((i + 1) + "." + taskList[i].toString());
-        }
-        print(Messages.LIST.MESSAGE, taskListString.toString());
-    }
-
     public static void main(String[] args) {
-        print(Messages.START.MESSAGE);
+        TaskList taskList = new TaskList();
 
+        Ui.printStartMessage();
         try (Scanner in = new Scanner(System.in)) {
             boolean isRunning = true;

```

Comment:
I like how the variable is named. it is very clear that that it is a boolean and it is in camelCase